src/js/legacy/mega-menu.js

```diff
@@ -67,11 +67,13 @@ class MegaMenu {
 
   closeMenu(event) {
     const target = event.target;
-    const menu = target.getAttribute('aria-controls');
+    const dropdown = this.getMenu(target.getAttribute('aria-controls'));
 
     event.stopPropagation();
     target.setAttribute('aria-expanded', false);
-    this.getMenu(target.getAttribute('aria-controls')).setAttribute('hidden','hidden');
+    dropdown.setAttribute('hidden','hidden');
+    
+    this.menu.classList.remove('vetnav--submenu-expanded');
   }
```

Comment:
The submenu's new style hides the parent menu items, so there needs to be a top-level CSS class added in order to apply that style. I didn't see a more elegant way than to just add that logic here.